packages/twenty-server/src/modules/workflow/workflow-executor/types/workflow-step-settings.type.ts

```diff
@@ -10,11 +10,16 @@ type BaseWorkflowStepSettings = {
 };
 
 export type WorkflowCodeStepSettings = BaseWorkflowStepSettings & {
-  serverlessFunctionId: string;
+  input: {
+    serverlessFunctionId: string;
+  };
 };
 
 export type WorkflowSendEmailStepSettings = BaseWorkflowStepSettings & {
-  connectedAccountId: string;
-  subject?: string;
-  body?: string;
+  input: {
+    connectedAccountId: string;
+    email: string;
+    subject?: string;
+    body?: string;
+  };
```

Comment:
Putting the input's configuration in a separate property is excellent. Way more clean! 👍 💯